docs/configuration/overview.mdx

```diff
@@ -88,6 +91,22 @@ ignore:
 > You can also expand the scope of ignored packages on a per-command basis via
 > the [`--scope` filter](/filters#scope) flag.
 
+## `dependencyOverrides`
```

Comment:
Should we move this to `command/bootstrap/dependencyOverrides` since this is specific to the bootstrap command?

Reply:
Good point!